src/services/user/firstLogin.js

```diff
@@ -33,15 +41,9 @@ const firstLogin = async (data, params, app) => {
 
 	// wrong birthday object?
 	if (data.studentBirthdate) {
-		data.studentBirthdate = moment(data.studentBirthdate).format('DD.MM.YYYY');
-		const dateArr = data.studentBirthdate.split('.');
-		const userBirthday = new Date(`${dateArr[1]}.${dateArr[0]}.${dateArr[2]}`);
-		// eslint-disable-next-line no-restricted-globals
-		if (userBirthday instanceof Date && isNaN(userBirthday)) {
-			return Promise.reject(new Error('Bitte einen validen Geburtstag auswählen.'));
-		}
-		userUpdate.birthday = userBirthday;
+		userUpdate.birthday = parseDate(data.studentBirthdate);
```

Comment:
the api should only accept timestamps+timezone or iso-strings with timezone info. parsing with a predefined (manually defined) format should not be used here. therefore i would suggest to update the data which is sent from the client to the server to be an iso string.

Reply:
i absolutely agree. but as this is an urgent hotfix in the legacy code we have to keep things as simple as possible. we will do things better in the future.